Vulkan-Pipeline-Assistant/descriptors.h

```diff
@@ -51,7 +53,7 @@ namespace vpa {
         const QMap<ShaderStage, PushConstantInfo>& PushConstants() const { return m_pushConstants; }
 
         void WriteBufferData(uint32_t set, int index, size_t size, size_t offset, void* data);
-        void LoadImage(int index, QString name);
+        void LoadImage(const uint32_t set, const int index, const QString name);
```

Comment:
Would it not be preffered to use uint32_t or some other form of unsigned integer for indices.

Reply:
It would be preferred. However, Qt decided their containers would be sized with ints, so I could have a uint32_t parameter but it will just be cast anyway when indexing the container in the function. I dislike this also.

Reply:
Acknowledged, in that case, concern is closed.